### DISCUSSION TITLE
Secondary panel at bottom?

### DISCUSSION THREAD

**Discussion by srhrnkn** [posit-dev/positron#3744] [Q&A] [answered]
Is it possible to arrange the panels so that the editor panel is at the top left, the console at the top right, and the "secondary" panel with vars/data/plots on the bottom? I prefer not to have the console cursor exiled to the far bottom of the screen.

**Comment by petetronic** [comment, marked as answer]
Maybe this will get you close to the above?

To get the Side-By-Side layout:
1. Open the Command Palette (i.e. on Mac Cmd+Shift+P)
2. Type "Customize Layout"
3. This should open a quick pick, select Side-By-Side Layout, placing Editor on left, the Console on the right.

To adjust the Panel:
  4. Open the Command Palette again
  5. Type "View: Set Panel Alignment to Justify" (if you want the Panel to take up the entire bottom section)
  6. Drag the Variables View from secondary side bar to dock with the other tabs on the Panel in the order you want
  7. Drag the Plots View  from secondary side bar to dock with the other tabs on the Panel in the order you want


Notes: 
For an alternative to 1. & 2., there is a toolbar icon in the top right of the screen to quickly choose Customize Layout. 
For an alternative to 5., you can Ctrl click on the Panel and in the pop-up menu go to  Align Panel, and select Justify.

**Comment by srhrnkn** [comment]
Yes! this did it. Thanks so much.
